modules/consul-cluster/main.tf

```diff
@@ -29,6 +29,8 @@ resource "aws_autoscaling_group" "autoscaling_group" {
   wait_for_capacity_timeout = "${var.wait_for_capacity_timeout}"
   service_linked_role_arn   = "${var.service_linked_role_arn}"
 
+  enabled_metrics = ["${var.enabled_metrics}"]
```

Comment:
What is the default value in `aws_autoscaling_group` if you don't specify this variable?

Reply:
Great question @brikis98 

The default value is an empty list, which is what I attempted to set the default value for `var.enabled_metrics` to, to match it. Confirmed when I switch back from my module fork (where I specified `enabled_metrics`) to the original unmodified module and run a plan:

```
Initializing modules...
- module.consul_cluster
  Getting source "github.com/hashicorp/terraform-aws-consul//modules/consul-cluster?ref=v0.4.1"
...
------------------------------------------------------------------------

An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  ~ module.consul_cluster.aws_autoscaling_group.autoscaling_group
      enabled_metrics.#:          "8" => "0"
      enabled_metrics.119681000:  "GroupStandbyInstances" => ""
      enabled_metrics.1940933563: "GroupTotalInstances" => ""
      enabled_metrics.308948767:  "GroupPendingInstances" => ""
      enabled_metrics.3267518000: "GroupTerminatingInstances" => ""
      enabled_metrics.3394537085: "GroupDesiredCapacity" => ""
      enabled_metrics.3551801763: "GroupInServiceInstances" => ""
      enabled_metrics.4118539418: "GroupMinSize" => ""
      enabled_metrics.4136111317: "GroupMaxSize" => ""


Plan: 0 to add, 1 to change, 0 to destroy.
```

Reply:
Got it, thanks!